shapely/geometry/base.py

```diff
@@ -476,25 +476,40 @@ def normalize(self):
         """
         return shapely.normalize(self)
 
-    # Binary operations
-    # -----------------
+    # Binary (overlay) operations
```

Comment:
```suggestion
    # Overlay operations
```

(binary does not seem helpful in this context)